tests/exporter.js

```diff
@@ -42,7 +42,10 @@ define([
                         done();
                     }
                 },
-                features: {rich_text: false},
+                features: {
+                    rich_text: false,
+                    case_micro_image: true
```

Comment:
Can you share why was this changed?
(Not well versed with vellum tests so hard to put this all together)

Reply:
Please refer to [this](https://github.com/dimagi/Vellum/pull/1108#discussion_r1481566734) comment. Essentially, this was added to mock the feature flag being enabled for the test.